mappings/com/mojang/blaze3d/platform/FramebufferInfo.mapping

```diff
@@ -1,4 +1,17 @@
-CLASS net/minecraft/class_4492 com/mojang/blaze3d/platform/FramebufferInfo
-	FIELD field_20457 target I
-	FIELD field_20458 renderBufferTarget I
-	FIELD field_20460 attachment I
+CLASS net/minecraft/class_4492 com/mojang/blaze3d/platform/FrameBufferInfo
+	COMMENT Contains global constants for the frame buffer and frame buffer states,
+	COMMENT normalized to the current implementation running on the target system.
+	FIELD field_20457 FRAME_BUFFER I
+	FIELD field_20458 RENDER_BUFFER I
+	FIELD field_20459 COLOR_ATTACHMENT I
+	FIELD field_20460 DEPTH_ATTACHMENT I
+	FIELD field_20461 FRAME_BUFFER_COMPLETE I
+		COMMENT {@see GL30#GL_FRAMEBUFFER_COMPLETE}
+	FIELD field_20462 FRAME_BUFFER_INCOMPLETE_ATTACHMENT I
+		COMMENT {@see GL30#GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT}
+	FIELD field_20463 FRAME_BUFFER_INCOMPLETE_MISSING_ATTACHMENT I
+		COMMENT {@see GL30#GL_FRAMEBUFFER_INCOMPLETE_MISSING_ATTACHMENT}
+	FIELD field_20464 FRAME_BUFFER_INCOMPLETE_DRAW_BUFFER I
+		COMMENT {@see GL30#GL_FRAMEBUFFER_INCOMPLETE_DRAW_BUFFER}
+	FIELD field_20465 FRAME_BUFFER_INCOMPLETE_READ_BUFFER I
+		COMMENT {@see GL30#GL_FRAMEBUFFER_INCOMPLETE_READ_BUFFER}
```

Comment:
fyi for see tags we don't wrap in `{}`

Reply:
I thought you had to for them to work?

Reply:
Going to leave this open until I can actually test that.

Reply:
As far as I know `@see` tags are grouped like in the end of jd with `@param` `@throws` `@return` etc.